src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -63,6 +65,9 @@ public class PPLTool implements Tool {
 
     public static final String TYPE = "PPLTool";
 
+    public static final List<String> SYSTEM_INDICES = List
+        .of(ML_MODEL_GROUP_INDEX, ML_MODEL_INDEX, ML_TASK_INDEX, ML_CONNECTOR_INDEX, ML_CONFIG_INDEX, ML_MODEL_CONTROLLER_INDEX);
```

Comment:
There could be more indices like: agent, memory etc, using regex like `.plugins-ml-*` is more flexible to adapt to future changes. BTW, do we need to consider other system indices which are not ml-common's?

Reply:
I think so. Is there a pattern to match all system indices?

Reply:
Looks like all system indices are starting with `.`, here are some examples: `.opendistro-alerting-config`,`.opensearch-observability`, `.replication-metadata-store`. Check if starting with `.` could be safe and also fine-tune the error message.

Reply:
Already done.